### DISCUSSION TITLE
Get error : "Response is too large" with "http.get" command

### DISCUSSION THREAD

**Discussion by Axelouuu-dev** [cc-tweaked/CC-Tweaked#1460] [Q&A] [answered]
Hi !

I try to download a file from my website but it give me the error "Response is too large".

In computercraft-server.toml config file, I've tried to up space_limit to 1Go, max_request to 0 and brandwidth to 2147483647.

The file I try to download weighs 17Mo.

![image](https://github.com/cc-tweaked/CC-Tweaked/assets/19538750/2eee7c36-93f2-42d3-95bc-5ccdb735b658)

Is there possible to increase the limit ?

**Comment by SquidDev** [comment, marked as answer]
There should be an option called "Max download" under the `http.rules` section of the config:

```toml
#The maximum size (in bytes) that a computer can download in a single request.
#Note that responses may receive more data than allowed, but this data will not
#be returned to the client.
max_download = 16777216
```

**Comment by Axelouuu-dev** [comment]
Alright, I change value to **167772160** and it works ! 

Thank :)
